mie-lab/trackintel
# TST: First test and update for postgis IO

supposed to fix #167 (work in progress)

In this pull request I already updated `write_triplegs_postgis` and I started with `read_triplegs_postgis` but I did not finish it yet. There are many open points left. 
In particular the following points are not yet adressed: 
- update all other postgis i/o functions
- Test all postgis i/o functions
- Update docstrings
- Ensure that test location and naming follows our convention
